### ISSUE TITLE
problems about using PrefetchDataset with batchsize better than 1 in "test.py"

### ISSUE THREAD

**Issue by charliess123** [xingyizhou/CenterNet#656]
Hi, I just try testing the trained ctdet model with the "test.py" you offer. And I want to use the "prefetch data function". 

But when I set the batch size from 1 to 10 in here. 
`  data_loader = torch.utils.data.DataLoader(
    PrefetchDataset(opt, dataset, detector.pre_process), 
    batch_size=10, shuffle=False, num_workers=1, pin_memory=True) `
I find out that the number of the result only leave 1/10.

Would you mind giving me some advices about how to testing with batch size bigger than 1?

**Comment by xingyizhou** [comment]
Unfortunately, we only support batch size 1 for testing.
